activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Support transforming empty-ish `has_many_attached` value into `[]` (e.g. `[""]`)
```

Comment:
@rafaelfranca after rebasing following the 7.0.0-rc1 release, this changelog entry is the only one in the file. Is this still the best place to mention this change?

Reply:
I'll move it to 7-0-stable. I should have included this feature in 7.0.0.rc1

Reply:
That sounds good to me. Is there anything I can do in this commit to make that easier for you?